lib/kovid/tablelize.rb

```diff
@@ -223,6 +239,31 @@ def align_columns(table_type, table)
         end
         table
       end
+
+      def top_row(data, options)
+        location_title = if options[:location] == :countries
+                           country_title(data)
+                         else
+                           data['state'].upcase
+                         end
+        [
+          location_title,
+          Kovid.comma_delimit(data['cases']),
+          Kovid.comma_delimit(data['deaths'])
+        ]
+      end
+
+      def top_heading(options)
+        return COUNTRY_CASES_DEATHS if options[:location] == :countries
```

Comment:
I'd love if you used a ternary operator here instead. What do you think? :)

Reply:
First, I used ternary operator. But, I got a long line. So, for sake of readability, I turned it to this.

Reply:
It becomes multi-line ternary operator. So, I prefer using `if` in a guard clause.